test/runnable/testaa2.d

```diff
@@ -69,7 +81,6 @@ void foo2()
 void testaa()
 {
     size_t i = foo("abc");
-    printf("i = %zd\n", i);
```

Comment:
Removed this `printf` because `z` (`size_t`) is not supported on Azure-mingw (and the assert below verifies i anway)